### PR TITLE
Capture name

### PR DESCRIPTION
**Pull request by Aupajo**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,9 +18,46 @@
...
+        border-top-right-radius: 2em;
+        border-bottom-right-radius: 2em;
+      }
     </style>
   </head>
   <body>
+    <form onsubmit="this.style.display = 'none'; Client.start(name.value); return false">
+      <input placeholder="Your name" id="name" onkeyup="this.value = this.value.replace(/[^A-Za-z]/, ''); this.value.length ? go.removeAttribute('disabled') : go.setAttribute('disabled', true)" maxlength="10" autofocus="true">
+      <input type="submit" value="Launch!" id="go" disabled>
+    </form>
```
Comment (Aupajo) [inline index.html]: Check out my sweet `onkeyup` validation.

Comment (kaleworsley) [inline index.html]: My sister has an apostrophe in her name.

Comment (kaleworsley) [inline index.html]: Hyphens are also a thing.

Comment (Aupajo) [inline index.html]: Nah. 10 letters of any case ought to be enough for anyone.

**Pull request merged**